apps/api/src/test-data/buildSpatialFeatureTestData/buildPolygonTestData.ts

```diff
@@ -30,8 +30,8 @@ export default (): Polygon[] =>
                 id: `${index + 300}`,
                 published: true,
                 properties: {
-                    name: `Point with ID: ${`${index + 300}`} Name`,
-                    description: `Description for point ${`${index + 300}`}`,
+                    name: `Traditional Deer Hunting Area`,
```

Comment:
Less generic

Reply:
Don't do this. The point is for us to know which piece of dummy data this is the name for. If you want to hard wire some realistic looking data, do this in addition to mechanically generated data. Also, as a general rule, you shouldn't touch existing dummy data, as some of our assertions in Cypress tests are relying on the details of this.